src/source/api-reference.rst

```diff
@@ -43,10 +43,11 @@ API Reference
    reference/post-metrics-data
    reference/campaign-status
    reference/post-campaign
-   reference/get-campaign
+   reference/get-campaigns
```

Comment:
not worth updating this, the whole file should be removed and not referenced anywhere. (I did remove it from index iirc)